presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -1355,6 +1355,64 @@ VeloxQueryPlanConverterBase::toVeloxQueryPlan(
       sourceVeloxPlan);
 }
 
+std::shared_ptr<const core::TableWriteNode>
+VeloxQueryPlanConverterBase::toVeloxQueryPlan(
+    const std::shared_ptr<const protocol::DeleteNode>& node,
+    const std::shared_ptr<protocol::TableWriteInfo>& tableWriteInfo,
+    const protocol::TaskId& taskId) {
+  std::string connectorId;
+  std::shared_ptr<connector::ConnectorInsertTableHandle> connectorInsertHandle;
+  if (
+      auto deleteHandle = std::dynamic_pointer_cast<protocol::DeleteHandle>(
```

Comment:
Nit maybe abstract a toConnectorDeleteTableHandle method like other functions (e.g. toConnectorTableHandle) in this file.

Reply:
Thanks for taking a look!  I added a for toVeloxQueryPlan(DeleteNode). It was a bit complex due to not having an existing connector actually implementing the delete portions, but I think I've successfully mocked up what was needed.